ee/frontend/mobile-replay/transform.test.ts

```diff
@@ -0,0 +1,303 @@
+            ])
+        })
+
+        test('can ignore unknown wireframe types', () => {
+            const unexpectedWireframeType = posthogEEModule.mobileReplay?.transformToWeb([
+                {
+                    data: { screen: 'App Home Page', width: 300, height: 600 },
+                    timestamp: 1,
+                    type: 4,
+                },
+                {
+                    type: 2,
+                    data: {
+                        wireframes: [
+                            {
+                                id: 12345,
+                                x: 11,
+                                y: 12,
+                                width: 100,
+                                height: 30,
+                                type: 'something in the SDK but not yet the transformer',
+                            },
+                        ],
+                    },
+                    timestamp: 1,
+                },
+            ])
+            expect(unexpectedWireframeType).toMatchSnapshot()
+        })
+
+        test('can short-circuit non-mobile full snapshot', () => {
+            const allWeb = posthogEEModule.mobileReplay?.transformToWeb([
+                {
+                    data: { href: 'https://my-awesome.site', width: 300, height: 600 },
+                    timestamp: 1,
+                    type: 4,
+                },
+                {
+                    type: 2,
+                    data: {
+                        node: { the: 'payload' },
+                    },
+                    timestamp: 1,
+                },
+            ])
+            expect(allWeb).toMatchSnapshot()
+        })
+
+        test('can convert images', () => {
+            const exampleWithImage = posthogEEModule.mobileReplay?.transformToWeb([
+                {
+                    data: {
+                        screen: 'App Home Page',
+                        width: 300,
+                        height: 600,
+                    },
+                    timestamp: 1,
+                    type: 4,
+                },
+                {
+                    type: 2,
+                    data: {
+                        wireframes: [
+                            {
+                                id: 12345,
+                                x: 11,
+                                y: 12,
+                                width: 100,
+                                height: 30,
+                                // clip: {
+                                //   bottom: 83,
+                                //   right: 44,
+                                // },
+                                type: 'text',
+                                text: 'Ⱏ遲䩞㡛쓯잘ጫ䵤㥦鷁끞鈅毅┌빯湌Თ',
+                                style: {
+                                    // family: '疴ꖻ䖭㋑⁃⻋ꑧٹ㧕Ⓖ',
+                                    // size: 4220431756569966319,
+                                    color: 'red',
```

Comment:
`color` is always `#RGB`

Reply:
Will you send "#12345" or "12345"? (for now I pass it straight through and hope it's a css valid color)

Reply:
Always as eg `#0000ff`